dev-resources/template/2_lrs.yml

```diff
@@ -515,7 +519,8 @@ Resources:
                     "webserver": {
                       "httpHost": "0.0.0.0",
                       "httpPort": ${InstanceHttpPort},
-                      "allowedOrigins": ${AllowedOrigins}
+                      "allowedOrigins": ${AllowedOrigins},
+                      "jwtCommonSecret": ${JWTCommonSecret}
```

Comment:
This is for sure not gonna work. This would just take the literal path that get input and print it, take a look here:

https://github.com/yetanalytics/lrsql/blob/main/dev-resources/template/1_db.yml#L127

Unfortunately it's possible you will need to get a lambda to decrypt it, or optionally we can just forget about SSM storage because it's going to be a complete PITA to decrypt. Try doing what was done in the other template first though.

Reply:
Yet another option is to NOT store it as a secure string, but to store it in SSM as a parameter. In that case it will be writable to this file via reference. I'm pretty sure SSM with secure strings needs a lambda decrypter which is complicated for any of us to do. @milt thoughts?

Reply:
see slack.